[openblas] Build error on x64-linux

Package: openblas:x64-linux@0.3.27

**Host Environment**

- Host: x64-linux
- Compiler: GNU 14.1.1
-    vcpkg-tool version: 2024-04-23-d6945642ee5c3076addd1a42c331bbf4cfc97457
    vcpkg-scripts version: bd2b54836 2024-05-14 (13 hours ago)

**To Reproduce**

`vcpkg install `

**Failure logs**

```
-- Using cached OpenMathLib-OpenBLAS-v0.3.27.tar.gz.
-- Cleaning sources at /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/michal/projects/sources/broom/vcpkg/downloads/OpenMathLib-OpenBLAS-v0.3.27.tar.gz
-- Applying patch uwp.patch
-- Applying patch fix-redefinition-function.patch
-- Applying patch install-tools.patch
-- Using source at /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean
-- Found external ninja('1.12.1').
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/cmake --build . --config Debug --target install -- -v -j25
    Working Directory: /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg
    See logs for more information:
      /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  /home/michal/projects/sources/broom/out/build/ninja_debug_vcpkg/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  /home/michal/projects/sources/broom/out/build/ninja_debug_vcpkg/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/openblas/portfile.cmake:72 (vcpkg_cmake_install)
  scripts/ports.cmake:175 (include)



```

<details><summary>/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/install-x64-linux-dbg-out.log</summary>

```
Change Dir: '/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg'

Run Build Command(s): /usr/bin/ninja -v -v -j25 install
[1/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/zcopy.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/zcopy.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/zcopy.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/zcopy.c
[2/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/ccopy.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/ccopy.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/ccopy.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/ccopy.c
[3/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/drot.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/drot.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/drot.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/drot.c
[4/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/dcopy.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/dcopy.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/dcopy.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/dcopy.c
[5/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/znrm2.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/znrm2.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/znrm2.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/znrm2.c
[6/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/sasum.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/sasum.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/sasum.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/sasum.c
[7/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/dasum.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/dasum.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/dasum.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/dasum.c
[8/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/dsum.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/dsum.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/dsum.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/dsum.c
[9/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/sdot.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/sdot.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/sdot.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/sdot.c
[10/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/dnrm2.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/dnrm2.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/dnrm2.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/dnrm2.c
[11/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/srot.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/srot.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/srot.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/srot.c
[12/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/sscal.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/sscal.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/sscal.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/sscal.c
[13/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/snrm2.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/snrm2.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/snrm2.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/snrm2.c
[14/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/sswap.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/sswap.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/sswap.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/sswap.c
[15/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/cnrm2.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/cnrm2.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/cnrm2.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/cnrm2.c
[16/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/saxpy.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/saxpy.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/saxpy.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/saxpy.c
[17/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/srotg.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/srotg.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/srotg.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/srotg.c
[18/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/srotm.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/srotm.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/srotm.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/srotm.c
[19/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/drotm.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/drotm.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/drotm.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/drotm.c
[20/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/drotmg.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/drotmg.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/drotmg.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/drotmg.c
[21/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/srotmg.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/srotmg.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/srotmg.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/srotmg.c
[22/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/ssum.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/ssum.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/ssum.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/ssum.c
[23/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/saxpby.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/saxpby.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/saxpby.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/saxpby.c
[24/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/scopy.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/scopy.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/scopy.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/scopy.c
[25/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -g -MD -MT interface/CMakeFiles/interface.dir/CMakeFiles/daxpy.c.o -MF interface/CMakeFiles/interface.dir/CMakeFiles/daxpy.c.o.d -o interface/CMakeFiles/interface.dir/CMakeFiles/daxpy.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg/interface/CMakeFiles/daxpy.c
...
Skipped 2049 lines
...
[1925/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xzcblat2.dir/c_zblat2c.c.o -MF ctest/CMakeFiles/xzcblat2.dir/c_zblat2c.c.o.d -o ctest/CMakeFiles/xzcblat2.dir/c_zblat2c.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat2c.c
[1926/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xccblat3_3m.dir/c_cblat3c_3m.c.o -MF ctest/CMakeFiles/xccblat3_3m.dir/c_cblat3c_3m.c.o.d -o ctest/CMakeFiles/xccblat3_3m.dir/c_cblat3c_3m.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_cblat3c_3m.c
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_cblat3c_3m.c: In function ‘main’:
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_cblat3c_3m.c:278:34: warning: initializer-string for array of ‘char’ is too long
  278 |     static char snames[9][13] = {"cblas_cgemm3m ", "cblas_chemm ", "cblas_csymm ",
      |                                  ^~~~~~~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_cblat3c_3m.c:278:34: note: (near initialization for ‘snames[0]’)
[1927/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xzcblat3.dir/c_zblat3c.c.o -MF ctest/CMakeFiles/xzcblat3.dir/c_zblat3c.c.o.d -o ctest/CMakeFiles/xzcblat3.dir/c_zblat3c.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat3c.c
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "photobroom",
  "version-string": "1.7.0",
  "port-version": 1,
  "homepage": "https://github.com/Kicer86/photobroom",
  "description": "Photo Broom is a tool for managing your photos.",
  "dependencies": [
    {
      "name": "dlib",
      "default-features": false,
      "features": [
        "fftw3"
      ]
    }
  ]
}

```
</details>

Actual error was not included in log above:
```
[1902/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xccblat3_3m.dir/c_xerbla.c.o -MF ctest/CMakeFiles/xccblat3_3m.dir/c_xerbla.c.o.d -o ctest/CMakeFiles/xccblat3_3m.dir/c_xerbla.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_xerbla.c
[1903/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o -MF ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o.d -o ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c
FAILED: ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o 
/usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o -MF ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o.d -o ctest/CMakeFiles/xzcblat1.dir/c_zblat1c.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c: In function ‘check1_’:
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:411:35: warning: suggest parentheses around ‘+’ inside ‘<<’ [-Wparentheses]
  411 |                 i__3 = i__ + (np1 + combla_1.incx * 5 << 3) - 49;
      |                               ~~~~^~~~~~~~~~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:425:41: error: passing argument 2 of ‘zscaltest_’ from incompatible pointer type [-Wincompatible-pointer-types]
  425 |                 zscaltest_(&combla_1.n, &ca, cx, &combla_1.incx);
      |                                         ^~~
      |                                         |
      |                                         doublecomplex *
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:383:54: note: expected ‘doublereal *’ {aka ‘double *’} but argument is of type ‘doublecomplex *’
  383 |     extern /* Subroutine */ int zscaltest_(integer*, doublereal*, doublecomplex*, integer*), itest1_(integer*, integer*), stest1_(doublereal*, doublereal*, doublereal*, doublereal*);
      |                                                      ^~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:426:47: warning: suggest parentheses around ‘+’ inside ‘<<’ [-Wparentheses]
  426 |                 ctest_(&len, cx, &ctrue5[(np1 + combla_1.incx * 5 << 3) - 48],
      |                                           ~~~~^~~~~~~~~~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:427:39: warning: suggest parentheses around ‘+’ inside ‘<<’ [-Wparentheses]
  427 |                          &ctrue5[(np1 + combla_1.incx * 5 << 3) - 48], sfac);
      |                                   ~~~~^~~~~~~~~~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:431:47: warning: suggest parentheses around ‘+’ inside ‘<<’ [-Wparentheses]
  431 |                 ctest_(&len, cx, &ctrue6[(np1 + combla_1.incx * 5 << 3) - 48],
      |                                           ~~~~^~~~~~~~~~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:432:39: warning: suggest parentheses around ‘+’ inside ‘<<’ [-Wparentheses]
  432 |                          &ctrue6[(np1 + combla_1.incx * 5 << 3) - 48], sfac);
      |                                   ~~~~^~~~~~~~~~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:459:27: error: passing argument 2 of ‘zscaltest_’ from incompatible pointer type [-Wincompatible-pointer-types]
  459 |         zscaltest_(&c__5, &ca, cx, &combla_1.incx);
      |                           ^~~
      |                           |
      |                           doublecomplex *
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:383:54: note: expected ‘doublereal *’ {aka ‘double *’} but argument is of type ‘doublecomplex *’
  383 |     extern /* Subroutine */ int zscaltest_(integer*, doublereal*, doublecomplex*, integer*), itest1_(integer*, integer*), stest1_(doublereal*, doublereal*, doublereal*, doublereal*);
      |                                                      ^~~~~~~~~~~
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c: In function ‘check2_’:
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:650:41: error: passing argument 2 of ‘zaxpytest_’ from incompatible pointer type [-Wincompatible-pointer-types]
  650 |                 zaxpytest_(&combla_1.n, &ca, cx, &combla_1.incx, cy, &
      |                                         ^~~
      |                                         |
      |                                         doublecomplex *
/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_zblat1c.c:598:54: note: expected ‘doublereal *’ {aka ‘double *’} but argument is of type ‘doublecomplex *’
  598 |     extern /* Subroutine */ int zaxpytest_(integer*, doublereal*, doublecomplex*, integer*, doublecomplex*, integer*);
      |                                                      ^~~~~~~~~~~
[1904/1950] /usr/bin/cc  -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean -I/home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/x64-linux-dbg -fPIC  -DNEEDBUNDERSCORE  -DHAVE_C11 -Wall -m64 -mavx2 -mavx -msse -msse2 -msse3 -mssse3 -msse4.1 -fPIC -DSMALL_MATRIX_OPT -DNO_LAPACK -DNO_LAPACKE -DNO_WARMUP -DMAX_CPU_NUMBER=24 -DMAX_PARALLEL_NUMBER=1 -DMAX_STACK_ALLOC=2048 -DNO_AFFINITY -DVERSION="\"0.3.27\"" -DBUILD_SINGLE -DBUILD_DOUBLE -DBUILD_COMPLEX -DBUILD_COMPLEX16 -DADD_ -DCBLAS -g -MD -MT ctest/CMakeFiles/xccblat3.dir/c_cblas3.c.o -MF ctest/CMakeFiles/xccblat3.dir/c_cblas3.c.o.d -o ctest/CMakeFiles/xccblat3.dir/c_cblas3.c.o -c /home/michal/projects/sources/broom/vcpkg/buildtrees/openblas/src/v0.3.27-b066c33329.clean/ctest/c_cblas3.c
```

--- a/ports/openblas/gcc14.patch
+++ b/ports/openblas/gcc14.patch
@@ -1,0 +1,19 @@
+Index: cmake/system.cmake
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
+===================================================================
+diff --git a/cmake/system.cmake b/cmake/system.cmake
+--- a/cmake/system.cmake	(revision ce3f668c992cb3cc80849d5c30ed637f5adbd5b2)
++++ b/cmake/system.cmake	(date 1718638483744)
+@@ -160,6 +160,10 @@
+   endif ()
+ endif ()
+ 
++if (${CMAKE_C_COMPILER_ID} STREQUAL "GNU")
++  set(CCOMMON_OPT "${CCOMMON_OPT} -Wno-error=incompatible-pointer-types")
++endif ()
++
+ include("${PROJECT_SOURCE_DIR}/cmake/prebuild.cmake")
+ if (DEFINED TARGET)
+   if (${TARGET} STREQUAL COOPERLAKE AND NOT NO_AVX512)

--- a/ports/openblas/portfile.cmake
+++ b/ports/openblas/portfile.cmake
@@ -8,6 +8,7 @@ vcpkg_from_github(
         uwp.patch
         fix-redefinition-function.patch
         install-tools.patch
+        gcc14.patch
 )
 
 find_program(GIT NAMES git git.cmd)

--- a/ports/openblas/vcpkg.json
+++ b/ports/openblas/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "openblas",
   "version": "0.3.27",
+  "port-version": 1,
   "description": "OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.",
   "homepage": "https://github.com/OpenMathLib/OpenBLAS",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6418,7 +6418,7 @@
     },
     "openblas": {
       "baseline": "0.3.27",
-      "port-version": 0
+      "port-version": 1
     },
     "opencascade": {
       "baseline": "7.8.1",

--- a/versions/o-/openblas.json
+++ b/versions/o-/openblas.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a11cdf69c9d9071c73f8d88458aa8b09c597d695",
+      "version": "0.3.27",
+      "port-version": 1
+    },
+    {
       "git-tree": "653ab9ede4c3c8b556b5d1b8b7d98d6abab8ef9d",
       "version": "0.3.27",
       "port-version": 0

